src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -2682,7 +2718,7 @@ TEST_P(conv_fp32_reorder_bfyx_to_fsv32_conv_data_padding, have_data_padding) {
     execute(p);
 }
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_reorder_bfyx_to_fsv32_conv_data_padding, ::testing::ValuesIn(std::vector<convolution_test_params>{
-    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  5, 6 }
+    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  5, 5 }
```

Comment:
Is it fused? ( it is mentioned that this "should be fused" above). If so, why the # of primtives are same b/w fused and unfused?

Reply:
'reorder_fsv32' is fused in the point without 'enable_reorder_fusing'. As the result, both fused and not-fused are fused.